admin_manual/configuration/occ_command.rst

```diff
@@ -0,0 +1,252 @@
+=====================
+Using the occ Command
+=====================
+
+ownCloud's ``occ`` command (ownCloud console) is ownCloud's command-line 
+interface. You can perform many common server operations with ``occ``::
+
+* Maintenance tasks
+* Manage apps
+* Upgrade the ownCloud database
+* Reset passwords, including administrator passwords
+* List files owned by users
+* Convert the ownCloud database from SQLite to a more performant DB
+* Query and change LDAP settings
+
+``occ`` is in the :file:`owncloud/` directory; for example 
+:file:`/var/www/owncloud` on Ubuntu Linux. ``occ`` is a PHP script. The 
+preferred way to run it is as your HTTP user. Running it with no options lists 
+all commands and options, like this example on Ubuntu:: 
+
+ $ sudo -u www-data php occ
+
+This is the same as ``sudo -u www-data php occ list``.
+ 
+Run it with the ``-h`` option for syntax help::
+
+ $ sudo -u www-data php occ -h
+ 
+Display your ownCloud version::
+
+ $ sudo -u www-data php occ -V
+   ownCloud version 7.0.4
+   
+Query your ownCloud server status::
+ 
+ $ sudo -u www-data php occ status
+   Array
+   (
+    [installed] => true
+    [version] => 7.0.4.2
+    [versionstring] => 7.0.4
+    [edition] => 
+   )
+   
+``occ`` has options, commands, and arguments. Options and arguments are 
+optional, while commands are required. The syntax is::
+
+ occ [options] command [arguments]
+ 
+Get detailed information on individual commands with the ``help`` command, like 
+this example for the ``maintenance:mode`` command::
+ 
+ $ sudo -u www-data php occ help maintenance:mode
+   Usage:
+   maintenance:mode [--on] [--off]
+
+   Options:
+   --on                  enable maintenance mode
+   --off                 disable maintenance mode
+   --help (-h)           Display this help message.
+   --quiet (-q)          Do not output any message.
+   --verbose (-v|vv|vvv) Increase the verbosity of messages: 1 for normal 
+   output, 2 for more verbose output and 3 for debug
+   --version (-V)        Display this application version.
+   --ansi                Force ANSI output.
+   --no-ansi             Disable ANSI output.
+   --no-interaction (-n) Do not ask any interactive question.
+   
+Maintenance Commands
+--------------------
+
+These three maintenance commands put your ownCloud server into three modes: maintenance, 
+singleuser, and repair.
```

Comment:
* Maintenance mode: Allows no user logins
* Single user mode: Allows just admin users to login (yes, I know - confusing name)
* repair: It's not a mode. It invokes the repair steps. https://github.com/owncloud/core/blob/3b2f33561bd7611ed9589a5d24513ad9e0803dbf/lib/private/repair.php#L79L85